src/dos/drive_overlay.cpp

```diff
@@ -304,9 +307,27 @@ static OverlayFile* ccc(DOS_File* file) {
 	return ret;
 }
 
-Overlay_Drive::Overlay_Drive(const char * startdir,const char* overlay, Bit16u _bytes_sector,Bit8u _sectors_cluster,Bit16u _total_clusters,Bit16u _free_clusters,Bit8u _mediaid,Bit8u &error)
-:localDrive(startdir,_bytes_sector,_sectors_cluster,_total_clusters,_free_clusters,_mediaid),special_prefix("DBOVERLAY") {
-	optimize_cache_v1 = true; //Try to not reread overlay files on deletes. Ideally drive_cache should be improved to handle deletes properly.
```

Comment:
Wow.. that's a truck load cleaned up! very nice.

Reply:
Yeah… someone had reaaaaaly wide monitor, I guess.

Funny thing - after I moved all members to be initialized in the initializer list, Clang detected `optimize_cache_v1` as unused variable (!). One more perk of following effc++ guidelines :)